phases/stack.go

```diff
@@ -1,9 +1,91 @@
 package phases
 
-import "fmt"
+import (
+	"io/ioutil"
+
+	"github.com/bitrise-io/bitrise/models"
+	"github.com/bitrise-io/go-utils/log"
+	"gopkg.in/yaml.v2"
+)
+
+var defaultStacks map[string]string = map[string]string{
+	"xamarin":      "osx-vs4mac-stable",
+	"cordova":      "osx-vs4mac-stable",
+	"react-native": "osx-vs4mac-stable",
+	"ionic":        "osx-vs4mac-stable",
+	"flutter":      "osx-vs4mac-stable",
+	"android":      "linux-docker-android",
+	"macos":        "osx-xcode-10.0",
+	"ios":          "osx-xcode-10.0",
+	"other":        "",
+}
+
+var optionsStacks = []string{
+	"linux-docker-android-lts",
```

Comment:
Note: Do we have any API call which could provide this list?

Reply:
I'm afraid we don't